x-pack/plugins/index_management/__jest__/client_integration/index_template_wizard/template_create.test.tsx

```diff
@@ -263,11 +263,11 @@ describe('<TemplateCreate />', () => {
         expect(find('stepTitle').text()).toEqual('Index settings (optional)');
       });
 
-      it('should not allow invalid json', async () => {
+      it.skip('should not allow invalid json', async () => {
```

Comment:
This test fails to meet the last assertion in this test in this PR: `form.getErrorsMessages()` just returns an empty array - i.e. it doesn't find any error messages.

However, I think this test is might actually not be testing what we think it does in `master` but due to "lucky" circumstances it just happens to pass anyway. In Node.js 12 however, the order of operations regarding Timers and the Microtask Queue [has changed](https://blog.insiderattack.net/new-changes-to-timers-and-microtasks-from-node-v11-0-0-and-above-68d112743eb3) and the test now actually breaks (at least that's my hypophysis).

`actions.completeStepThree` is an `async` function, but in this test we forget to `await` it. If we add the `await` keyword (as I've done below) in `master`, it will actually break with the exact same error as it breaks with (both with and without the `await` keyword) in this PR.

@elastic/es-ui Could I get you to verify this test actually works as expected in `master` and if not fix it there. If I'm wrong and the test _does_ work as expected in `master`, I could use your help in figuring out what's wrong with the test in this PR.